website/docs/deployment.mdx

```diff
@@ -85,6 +85,16 @@ export default function Home() {
 }
 ```
 
+## Protecting your site {#protecting-your-site}
+
+The following are a couple of potential ways to protect sensitive content on a Docusaurus site:
+
+1. Password protection, some static hosts and CDNs offer this service. For example, [Netlify](#deploying-to-netlify) provides a [password based]( https://docs.netlify.com/visitor-access/password-protection/) function for controlling content access.
+
+2. Cookie or OAuth workflow, CDNs with edge serverless functions support this feature set. For example, using [Cloudflare](#deploying-to-cloudflare-pages) workers an authentication workflow like [GitHub Oauth](https://github.com/gr2m/cloudflare-worker-github-oauth-login) can be implemented.
+
+3. A self service option, implementing client-side rendering routes and fetching content from a protected API, [Gatsby.js](https://www.gatsbyjs.com/docs/how-to/routing/client-only-routes-and-user-authentication) is an example of this pattern.
```

Comment:
While I'm okay to have the other two in the documentation (though not very happy to), this is a part that we should NOT include. Gatsby is a direct replacement for Docusaurus. We are interested in how _we_ can do it, not how _they_ do it.

> - `addRoute({path: "/admin/*",component: "@site/src/Admin"})`
> - Admin should statically render a spinner or login page on the server => `/admin/index.html`
> - add Netlify redirect from `/admin/*` to `/admin/index.html` (for all i18n languages)
> - Use client-side route components in Admin after mount
> - Use API with authentication to fetch data at runtime in Admin routes

This part is the most relevant in https://github.com/facebook/docusaurus/issues/958#issuecomment-1057075925, all the other content is actually just "references" or "status quo". However, this instruction is quite vague—you probably need to actually implement that first as POC, and from there we can discuss which parts to include in the documentation.

Reply:
@Josh-Cena in terms of implementing a POC that would probably be beyond me, in scoping this pull request as a documentation add would we want to add those first two items?
Or alternatively reverting this PR and bundling that POC and documentation edit in one PR.

Reply:
TBF, I'm not entirely sure how we would progress. The first two points can be sufficient for a lot of users, but without the third point (written in a clear and instructive way) the doc seems obviously incomplete. Let's hold it as this until I (or someone else) figure out a way to move forward.

Reply:
yes, having a POC seems useful otherwise we are advising things that we didn't even try 😅